users/views.py

```diff
@@ -49,7 +41,7 @@ def get(self, request, *args, **kwargs):
 
 class UserUpdateView(LoginRequiredMixin, UserPassesTestMixin, UpdateView):
     template_name = 'users/profile.html'
-    model = User
+    model = CustomUser
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import